lib/controller/page/type/page.summary/page.summary.controller.js

```diff
@@ -164,6 +164,40 @@ SummaryController.attachJsonSubmission = async (submissions, userData) => {
   submissions.push(submission)
 }
 
+SummaryController.attachCsvSubmission = async (submissions, userData) => {
+  function formatEmail (str, markdown = false) {
+    try {
+      str = format(str, userData.getScopedUserData(), {markdown, lang: userData.contentLang})
+    } catch (e) {
+      //
+    }
+    return str
+  }
+
+  const serviceName = getInstanceProperty('service', 'name')
+  const subject = formatEmail(
+    getInstanceProperty('service', 'emailSubjectTeam') || `${serviceName} submission`
+  )
+  const defaultEmailAddress = getString('email.address.from')
+  const from = defaultEmailAddress ? formatEmail(defaultEmailAddress) : '"Form Builder" <form-builder@digital.justice.gov.uk>'
+  const to = formatEmail(SERVICE_OUTPUT_EMAIL)
+
+  const submission = {
+    type: 'csv',
+    recipientType: 'team',
+    from: from,
```

Comment:
Variables of the same name as the field they are assigned to no longer need a colon, so:
```
const from = 'Jonathan Perry'
const submission = { 
  from
}
```
Is equivalent to (as you have it):
```
const from = 'Jonathan Perry'
const submission = { 
  from: from
}
```
But it's less typing ;)